rotkehlchen/globaldb/cache.py

```diff
@@ -58,18 +58,38 @@ def globaldb_get_general_cache_values(
         key_parts: Iterable[Union[str, GeneralCacheType]],
 ) -> list[str]:
     """Function to read globaldb cache. Returns all the values that are paired with the key."""
-    cache_key = _compute_cache_key(key_parts)
+    cache_key = compute_cache_key(key_parts)
     cursor.execute('SELECT value FROM general_cache WHERE key=?', (cache_key,))
     return [entry[0] for entry in cursor]
 
 
+def globaldb_get_general_cache_keys_and_values_like(
+        cursor: DBCursor,
+        key_parts: Iterable[Union[str, GeneralCacheType]],
+) -> list[tuple[str, str]]:
+    """
+    Function to read globaldb cache.
+    Returns all pairs key-value where key starts with the provided `key_parts`.
+    """
+    assert all(
```

Comment:
We never check validity of arguments like that with assertions. Add these in the docstring.